src/pykeen/models/unimodal/rgcn.py

```diff
@@ -24,15 +24,73 @@
 class RGCN(
     ERModel[torch.FloatTensor, RelationRepresentation, torch.FloatTensor],
 ):
-    """An implementation of R-GCN from [schlichtkrull2018]_.
+    r"""An implementation of R-GCN from [schlichtkrull2018]_.
 
-    This model uses graph convolutions with relation-specific weights.
+    The Relational Graph Convolutional Network (R-GCN) comprises two parts:
+
+    1. A GCN-based entity encoder that computes enriched representations for entities, cf.
+       :class:`pykeen.nn.emb.RGCNRepresentations`. The GCN is modified to use different weights depending on the
+       type of the relation.
+    2. An arbitrary interaction model which computes the plausibility of facts given the enriched representations, 
+       cf. :class:`pykeen.nn.modules.Interaction`.
+
+    # TODO: Move the following part to RGCNRepresentations
+
+    The GCN employed by the entity encoder is adapted to include typed edges.
+    The forward pass of the GCN is defined by:
+
+     .. math::
+
+        \textbf{e}_{i}^{l+1} = \sigma \left( \sum_{r \in \mathcal{R}}\sum_{j\in \mathcal{N}_{i}^{r}}
+        \frac{1}{c_{i,r}} \textbf{W}_{r}^{l} \textbf{e}_{j}^{l} + \textbf{W}_{0}^{l} \textbf{e}_{i}^{l}\right)
+
+    where $\mathcal{N}_{i}^{r}$ is the set of neighbors of node $i$ that are connected to
+    $i$ by relation $r$, $c_{i,r}$ is a fixed normalization constant (but it can also be introduced as an additional
+    parameter), and $\textbf{W}_{r}^{l} \in \mathbb{R}^{d^{(l)} \times d^{(l)}}$ and
+    $\textbf{W}_{0}^{l} \in \mathbb{R}^{d^{(l)} \times d^{(l)}}$ are weight matrices of the `l`-th layer of the
+    R-GCN.
+
+    The encoder aggregates for each node $e_i$ the latent representations of its neighbors and its
+    own latent representation $e_{i}^{l}$ into a new latent representation $e_{i}^{l+1}$.
+    In contrast to standard GCN, R-GCN defines relation specific transformations
+    $\textbf{W}_{r}^{l}$ which depend on the type and direction of an edge.
+
+    The interaction model computes the plausibility score given the node representations $\textbf{e}_{i}^{L}$ that are
+    computed by the last layer $L$ of the R-GCN, i.e., for a given triple $(h,r,t) \in \mathcal{K}$, the
+    corresponding node representations $h:=e_i^L$ and $t:=e_j^L$ are used:
+
+    .. math::
+
+        f(h,r,t) = \textbf{h} \textbf{R}_{r} \textbf{t}
```

Comment:
it's not written where `\textbf{R}_{r}` comes from

Reply:
From the implementation, I guess this is learned independently from the RGCN representations

Reply:
Yes, this is learned independently.